docs/2.guide/1.concepts/3.rendering.md

```diff
@@ -89,7 +89,8 @@ Nuxt 3 starting from rc.12 comes with the public beta for route rules and hybrid
 - `ssr` - Disables server-side rendering for sections of your app and make them SPA-only with `ssr: false`
 - `cors` - Automatically adds cors headers with `cors: true` - you can customize the output by overriding with `headers`
 - `headers` - Add specific headers to sections of your site - for example, your assets
-- `static` and `swr` - `static` enables a single (on-demand) build; `swr` enables a static build, that lasts for a configurable TTL. (currently enables full incremental static generation on Netlify, with Vercel coming soon)
+- `swr`: Server-side renders sections of your app and add cache headers in the response for the server or the reverse proxy to cache it for a configurable TTL. For Netlify and Vercel, the response is also added to the CDN layer. Note that the node-server preset of Nitro is able to cache the response.
```

Comment:
Node.js/Server is the default behaving and also works for any server preset including workers, etc we should move to the beginning and mention it is cached on storage layer.

Behavior of Vercel/Netlify presets that map swr to a similar implementation is exceptional part!

Reply:
Updated.